This patch makes sure that the xineplug_decode_ff plugin, that is used by xine
to decode streams using ffmpeg, links against libpostproc and does not rely on
the fact that it's pulled in from other libraries (libavcodec/libavutil).  This
behaviour breaks when using ffmpeg 20060302 or previous with --as-needed
LDFLAGS.

See Gentoo Bug #124802.

Index: xine-lib-1.1.1/src/libffmpeg/Makefile.am
===================================================================
--- xine-lib-1.1.1.orig/src/libffmpeg/Makefile.am
+++ xine-lib-1.1.1/src/libffmpeg/Makefile.am
@@ -2,7 +2,7 @@ include $(top_srcdir)/misc/Makefile.comm
 
 if HAVE_FFMPEG
 ff_cppflags = $(FFMPEG_CPPFLAGS)
-link_ffmpeg = $(FFMPEG_LIBS)
+link_ffmpeg = $(FFMPEG_LIBS) $(FFMPEG_POSTPROC_LIBS)
 else
 link_ffmpeg = \
   $(top_builddir)/src/libffmpeg/libavcodec/libavcodec.la \
